tasks/config.yml

```diff
@@ -9,7 +9,7 @@
     mode: u=rw,g=r,o=r
   become: true
   notify:
-    - reload frr
+    - restart frr
```

Comment:
why a restart of frr - a reload of frr should work here - don't want to restart frr every time there is a change if we can avoid it and just reload it.

Reply:
I'm not exactly aware of what is covered by reload.  I encountered an issue when the role was run for the first time on a clean machine. File /etc/frr/frr.conf didn't existed yet. Reload failed when flush handlers were triggered before it was configured https://github.com/mrlesmithjr/ansible-frr/blob/3a7ed429f4b4bc068eef8855cd6664b33879b79d/tasks/config.yml#L56

There is a comment, that `We need to rediscover facts if interfaces are initially configured in zebra.conf`, it is code added in commit https://github.com/mrlesmithjr/ansible-frr/commit/041af27f6ff1bc100f3607cd5be959b44d01e8b2#diff-f8020d83f5c78e15b9807b118f54493c15b8b217402b0c02fe644bc8a20202d4 by @mrlesmithjr 

We can enable reload on this task however, we need to pick options:
1.) create empty frr.conf before flushing handlers to prevent 
2.) do research about the necessity of flushing handlers and eventually remove it

Reply:
I read docs and find out info about [integrated-configuration-mode](http://docs.frrouting.org/en/latest/vtysh.html#integrated-configuration-mode) which we already use https://github.com/mrlesmithjr/ansible-frr/blob/master/templates/etc/frr/vtysh.conf.j2#L3

So zebra.conf is not read and its configuration may be removed and flush_handlers also. After that, I may use reload again.

Reply:
Okay let's try to minimize how much config we do in zebra and vtysh - if you want I can test and make some changes as well. But would like to reduce any risk of accidentally having this push restarts to frr hosts when there's a small change elsewhere.

Reply:
I'll do it. I agree with you on using reload instead of restart where it is possible. It just was hotfix to handle errors when testing role.

Reply:
@nertwork take o look at changes

Reply:
@nertwork can we move on?

Reply:
Yes - thank you!